packages/core/src/asciidoctor-core-api.js

```diff
@@ -3047,3 +3047,387 @@ Opal.Asciidoctor.SyntaxHighlighterBase = SyntaxHighlighterBase
+}
+
+/**
+ * Set the column span of this {@link Cell} node.
+ * @param {number} value
+ * @returns {number} - The new colspan value
+ * @memberof Cell
+ */
+Cell.prototype.setColumnSpan = function (value) {
+  return this['$colspan='](value)
+}
+
+/**
+ * Get the row span of this {@link Cell} node
+ * @returns {number} - An Integer of the number of rows this cell will span (default: undefined)
+ * @memberof Cell
+ */
+Cell.prototype.getRowSpan = function () {
+  var rowspan = this.rowspan
+  return rowspan === Opal.nil ? undefined : rowspan
+}
+
+/**
+ * Set the row span of this {@link Cell} node
+ * @param {number} value
+ * @returns {number} - The new rowspan value
+ * @memberof Cell
+ */
+Cell.prototype.setRowSpan = function (value) {
+  return this['$rowspan='](value)
+}
+
+/**
+ * Get the content of the cell.
+ * This method should not be used for cells in the head row or that have the literal style.
+ * @returns {string}
+ * @memberof Cell
+ */
+Cell.prototype.getContent = function () {
+  var content = this['$content']()
+  return content === Opal.nil ? undefined : content
+}
+
+/**
+ * Get the text of the cell.
+ * @returns {string}
+ * @memberof Cell
+ */
+Cell.prototype.getText = function () {
+  var text = this['$text']()
+  return text === Opal.nil ? undefined : text
+}
+
+/**
+ * Get the source of the cell.
+ * @returns {string}
+ * @memberof Cell
+ */
+Cell.prototype.getSource = function () {
+  var source = this['$source']()
+  return source === Opal.nil ? undefined : source
+}
+
+/**
+ * Get the lines of the cell.
+ * @returns {Array<string>}
+ * @memberof Cell
+ */
+Cell.prototype.getLines = function () {
+  var lines = this['$lines']()
+  return lines === Opal.nil ? undefined : lines
+}
+
+/**
+ * Get the line number of the cell.
+ * @returns {number}
+ * @memberof Cell
+ */
+Cell.prototype.getLineNumber = function () {
+  var lineno = this['$lineno']()
+  return lineno === Opal.nil ? undefined : lineno
+}
+
+/**
+ * Get the file of the cell.
+ * @returns {string}
+ * @memberof Cell
+ */
+Cell.prototype.getFile = function () {
+  var file = this['$file']()
+  return file === Opal.nil ? undefined : file
+}
+
+/**
+ * Get the style of the cell.
+ * @returns {string}
+ * @memberof Cell
+ */
+Cell.prototype.getStyle = function () {
+  var style = this['$style']()
+  return style === Opal.nil ? undefined : style
+}
+
+/**
+ * Get the column of this cell.
+ * @returns {Column}
+ * @memberof Cell
+ */
+Cell.prototype.getColumn = function () {
+  var column = this.$column()
+  return column === Opal.nil ? undefined : column
+}
+
+/**
+ * Get the width of this cell.
+ * @returns {string|undefined}
+ * @memberof Cell
+ */
+Cell.prototype.getWidth = function () {
+  return this.getAttribute('width')
+}
+
+/**
+ * Get the column width in percentage of this cell.
+ * @returns {string|undefined}
+ * @memberof Cell
+ */
+Cell.prototype.getColumnPercentageWidth = function () {
+  return this.getAttribute('colpcwidth')
```

Comment:
@mojavelinux How do you feel about these utilities functions to "quickly" access the underlying attribute?